### PR TITLE
Low med priority bugs

### PR DESCRIPTION
**Pull request by plondon**

### CODE REVIEW

**Comment by Sjors** [comment]
Did you check if these fonts get included and are properly renamed with `grunt dist_unsafe`?

File: server.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ app.use(function (req, res, next) {
       "script-src 'self' 'sha256-mBeSvdVuQxRa2pGoL8lzKX14b2vKgssqQoW36iRlU9g=' 'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='",
       "connect-src 'self' " + rootURL + " " + (webSocketURL || "wss://*.blockchain.info") + " " + (apiDomain ||  "https://api.blockchain.info"),
       "object-src 'none'",
-      "media-src 'self' data: mediastream: blob:",
+      "media-src 'self' https://dl.dropboxusercontent.com/s/9qir4ihsyk2wn8d/blockchain-ad.mp4 https://dl.dropboxusercontent.com/s/irml3dze4i2q0ga/legacy-what-is-bitcoin-TEMP.mp4 data: mediastream: blob:",
```
Comment (plondon) [inline server.js]: verified grunt dist_unsafe as well

File: package.json [hunk 2]
```diff
@@ -45,7 +45,8 @@
     "karma-osx-reporter": "^0.1.0",
     "karma-phantomjs-launcher": "^0.1.4",
     "node-env-file": "^0.1.4",
-    "shelljs": "^0.3.0"
+    "shelljs": "^0.3.0",
+    "left-pad": "git://github.com/azer/left-pad.git#bff80e3ef0db0bfaba7698606c4f623433d14355"
```
Comment (plondon) [inline package.json]: https://github.com/azer/left-pad/issues/4

**Comment by jtormey** [comment]
ACK

**Pull request merged**